css/properties/animation.json

```diff
@@ -139,7 +139,10 @@
               "experimental": false,
               "standard_track": true,
               "deprecated": false
-            }
+            },
+            "tags": [
+              "web-features:scroll-driven-animations"
```

Comment:
This (css.properties.animation.animation-timeline_included) was listed in https://github.com/web-platform-dx/web-features/pull/582 but not in https://github.com/mdn/browser-compat-data/pull/22644 and seems like a simple oversight.